dencam/example_config.yaml

```diff
@@ -17,3 +17,5 @@ FRAME_RATE: 25
 
 # 0, 90, 180, 270. have not checked whether 90 and 270 crop image or what
 CAMERA_ROTATION: 180 
+
+AIRPLANEMODE: False
```

Comment:
I would venture this reads more easily as AIRPLANE_MODE. Maybe it might even be worth calling it START_IN_AIRPLANE_MODE?